UI/index.html

```diff
@@ -42,21 +42,30 @@
     <section class="container">
         <div class="one">
             <div class="side">
-                <img src="./images/download.jfif" /> <span>&nbsp;&nbsp;RWANDA</span><br /><br />
-                <a href="./pages/addRedflag.html" class="links">
-                    <img class="img1" src="./images/red-flag-vector-1253881 (1).jpg" />Redflag</a><br /><br />
-                <a href="./pages/addIntervention.html" class="links"><img class="img1"
-                        src="./images/red-button-help-icon-vector-19997793 (1).jpg" />Intervention</a><br /><br /><br /><br />
-                <p>Helpful links</p>
-                <div class="links1">
-                    <a href="#">About us</a><br /><br />
-                    <a href="#">Terms and conditions</a><br /><br />
-                    <a href="#">Privacy notices</a><br /><br />
-                    <a href="#">Contact us</a>
+                <div class="icon"><div class="rw"><img src="./images/download.jfif" /> &nbsp;<span>RWANDA</span></div><br /><br />
+                    
+                  <div class="incidents">
+                        <a href="./pages/aboutRedflag.html" class="links">
+                        <i class="far fa-flag"></i>&nbsp;&nbsp;Redflag</a>
+                  </div>
+                  <div class="incidents">
+
+                    <a href="./pages/aboutIntervention.html" class="links"><i
+                            class="fas fa-people-carry"></i>&nbsp;&nbsp;Intervention</a>
+                        </div>
+                        </div>
+                <div class="fo">
+                    <p>Helpful links</p>
+                    <div class="links1">
```

Comment:
``optional`` It would be nice if you use a ``ul`` or ``li` here.

Reply:
let me work on it now